src/Storages/MergeTree/MergeTreeData.h

```diff
@@ -393,6 +393,7 @@ class MergeTreeData : public IStorage, public WithMutableContext
     Block getMinMaxCountProjectionBlock(
         const StorageMetadataPtr & metadata_snapshot,
         const Names & required_columns,
+        bool has_filter,
```

Comment:
It is not clear what this flag is needed for. Could you please update a comment above?

Reply:
Done